bin/dbServer.py

```diff
@@ -31,49 +31,29 @@
 from flask import Flask, request
 import json
 import logging as log
-import os
 import sys
-
-import ConfigParser
-import sqlalchemy
-from sqlalchemy.engine.url import URL
-
 from lsst.dax.dbserv import dbREST_v0
+from lsst.db.engineFactory import getEngineFromFile
 
 app = Flask(__name__)
 
-def initEngine():
-    config = ConfigParser.ConfigParser()
-    defaults_file = os.path.expanduser("~/.lsst/dbAuth-dbServ.txt")
-    config.readfp(open(defaults_file))
-    db_config = dict(config.items("mysql"))
-    # Translate user name
-    db_config["username"] = db_config["user"]
-    del db_config["user"]
-    # SQLAlchemy part
-    url = URL("mysql",**db_config)
-    return sqlalchemy.create_engine(url)
-
-engine = initEngine()
-
+# Configure Engine
+defaults_file = "~/.lsst/dbAuth-dbServ.ini"
+engine = getEngineFromFile(defaults_file)
 app.config["default_engine"] = engine
 
-@app.route('/')
-def getRoot():
-    return '''Test server for testing db. Try adding /db to URI.
-'''
 
-@app.route('/db')
-def getDb():
-    '''Lists supported versions for /db.'''
+@app.route('/')
+def application_root():
```

Comment:
Actually the function names are inconsequential, I was just trying to do some PEP8 cleanup. The reason why they are inconsequential is because the decorator above the name, `@app.route`, registers the function at import time to the Flask framework. The Flask framework handles incoming HTTP requests, finds out routes match (`/db` in this case), and dispatches to the method. So Flask, in effect, is the only client to these methods and it doesn't really care about the method name (except for debugging purposes)

However, the HTTP route did, in fact, change. SUI is the main consumer of the HTTP interface and they've been notified. The routes changed to more closely adhere to the guidelines in the TAP interface.